packages/lab/src/badge/Badge.css

```diff
@@ -1,33 +1,38 @@
 /* Style applied to the root element */
 .saltBadge {
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
   display: inline-flex;
+  position: relative;
   flex-shrink: 0;
-  justify-content: center;
+  vertical-align: middle;
 }
 
 .saltBadge-badge {
   /* Should this vary according to touch size */
   padding-inline: var(--salt-spacing-50);
   line-height: var(--salt-accent-lineHeight);
+  height: var(--salt-accent-lineHeight);
   min-width: var(--salt-accent-lineHeight);
-  max-width: 50px;
   border-radius: 9999px;
+  white-space: nowrap;
+  z-index: var(--salt-zIndex-notification);
 
   align-items: center;
   justify-content: center;
-  display: inline;
+  display: inline-flex;
   margin: auto;
 
   font-size: var(--salt-accent-fontSize);
   font-weight: var(--salt-accent-fontWeight);
   background: var(--salt-accent-background);
   color: var(--salt-accent-foreground);
+
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
 }
 
 .saltBadge-topRight {
-  position: relative;
-  top: -7px;
-  right: 4px;
+  position: absolute;
+  top: 0;
+  right: 0;
+  transform: translate(50%, -50%);
```

Comment:
This positioning translates the badge by 50% of the badge size, so as the badge length varies with strings and numerical values the positioning isn't consistent. The way i found to achieve consistency was with relative positioning and displacing by a specified number of pixels.

Reply:
This positioning scales with density so arguably is better. I'm unsure whether consistency in positioning is an issue here. This needs design consideration anyway

Reply:
if we offset with one of the spacing vars from the theme, will that no also scale with density and keep the positioning more consistent independent of the content

Reply:
We've had some guidance from design. I'll ping you